gen/definitions/policyassignments.yaml

```diff
@@ -0,0 +1,66 @@
+---
+name: Policy Assignments
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/assignment/policyassignments
+no_delete: true
+no_data_source: true
+data_source_name_query: true
+doc_category: Policy Assignments
+attributes:
+  - model_name: type
+    type: String
+    value: "PolicyAssignment"
+  - model_name: name
+    type: String
+    description: User-created name of the resource.
```

Comment:
This is the name of the policy to be assigned, not name of the `fmc_policy_assignment` resource.

Reply:
Done